addons/l10n_in/models/account_move_line.py

```diff
@@ -48,7 +49,7 @@ class AccountMoveLine(models.Model):
             ("purchase_out_of_scope", "Out of Scope"),
             ],
         string="GSTR Section",
-        index=True,
+        index="btree_not_null",
```

Comment:
Nitpick: technical strings should be `'` not `"`. But the others are `"`, so I guess it's better to be consistent.